### PR TITLE
Add bert pretraining preprocessing

### PR DESCRIPTION
**Pull request by ji-xin**
## Description
 Please include a summary of the change and which issue is fixed.
 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist
- [x] My code follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Todo
- [ ] Please add WIP tasks in this section
- [ ] Ensure that the pull request title is of the format "WIP: Title" until all of these are checked
- [ ] Remove this section from the pull request description if no pending tasks exist

### CODE REVIEW

File: datasets/java_top1k/try.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import tqdm
```
Comment (achyudh) [inline datasets/java_top1k/try.py]: Do we really need this file?

**Review by achyudh — CHANGES_REQUESTED** [review]
Can you please remove datasets/java_top1k/try.py? It doesn't seem to do anything.

**Comment by ji-xin** [comment]
Deleted

**Review by achyudh — APPROVED** [review]

**Comment by achyudh** [comment]
LGTM. But I am squashing the changes because of the commit history.

**Pull request merged**